pages/docs/components/layout/stack.mdx

```diff
@@ -131,22 +136,16 @@ function StackEx() {
 render(<StackEx />)
 ```
 
-### Stack items horizontally
+#### Feature Cards with HStack Component
 
-Pass the `direction` and set it to `row`. Optionally, you can use `align` and
-`justify` to adjust the alignment and distribution of the items.
+In the example following, we can notice that for the `HStack` component, we
+don't need the direction props, since this component is specifically for
+horizontally stacking items.
```

Comment:
You could also mention that the Items are also aligned in the center by default :)